src/main/extension.cc

```diff
@@ -132,11 +132,11 @@ void leap_process_arguments(core::CommandLineOptions* options) {
              " -s         Ignore leaprc startup file.\n"
              " -I {dir}   Add {dir} to search path.\n"
              " -f {file}  Source {file}.\n"
-             " -t c       Startup using cclasp and quicklisp for all systems\n");
+             " -B         Startup using base image and quicklisp for all systems\n");
```

Comment:
Print here states to use `-B` for base image, but according to line 137, the option is still `-t`